templates/base/footer.tmpl

```diff
@@ -89,6 +89,37 @@
 		issuesTribute.attach(document.getElementById('content'))
 	</script>
 	{{end}}
+	<script>
+		var emojiTribute = new Tribute({
+			trigger: ':',
+			values: function (text, cb) {
+				var array = emojify.emojiNames;
+				var data = [];
+				var counter = 0;
+				for(var j=0; j<array.length; j++) {
```

Comment:
You can just use `j < Math.min(array.length, 5)` and than you don't need `counter` variable and condition below with break.

Reply:
I guess there should be 5 elements in the data array, which is not always the case in a loop to 5 because of the if condition.

Reply:
Yes, this loop is used to limit the amount of results to a maximum of 6 - but traverses all emojiNames using `j`. Removing the `counter` and replacing the condition with `data.length >5` looks much better to me. Thanks @daviian